modules/features/settings/src/main/java/au/com/shiftyjelly/pocketcasts/settings/whatsnew/WhatsNewViewModel.kt

```diff
@@ -56,8 +169,8 @@ class WhatsNewViewModel @Inject constructor(
     }
 
     sealed class WhatsNewFeature(
-        @StringRes val title: Int,
-        @StringRes val message: Int,
+        @StringRes open val title: Int,
+        @StringRes open val message: Int,
```

Comment:
If the only reason we're allowing these to be overridden is because of the name conflict with the `Bookmark` construction parameters, I'd probably lean toward leaving these as not open and instead using a slightly different name (i.e., `_title`) for the construction parameters in Bookmarks (which could then be private). Not a big deal though.